README.md

```diff
@@ -31,14 +69,393 @@ Permissions to allow the user to interact with the support center:
         }
     ]
 }
+
 ```
 
 
-## Testing your AWS account credentials with the CLI
+## 1.3. Workflow
+
+First, an AcccountPool must be created to specify the number of desired accounts to be ready. The operator then goes and creates that number of accounts. 
+When a [Hive](https://github.com/openshift/hive) cluster has a new cluster request, an AccountClaim is created with the desired name of the cluster in a unique namespace. The operator links the AccountClaim to an Account in the pool, and creates the required k8s secrets, placing them in the AccountClaim's unique namespace. The AccountPool is then filled up again by the operator.  Hive then uses the secrets to create the AWS resources for the new cluster. 
+
+For more information on how this process is done look at the controllers section.  
+
+
+## 1.4. Testing your AWS account credentials with the CLI
+
 The below commands can be used to test payer account credentials where we create new accounts inside the payer accounts organization. Once the account is created in the first step we wait until the account is created with step 2 and retrieve its account ID. Using the account ID we can then test our IAM user has sts:AssumeRole permissions to Assume the OrganizationAccountAccessRole in the new account. The OrganizationAccountAccessRole is created automatically when a new account is created under the organization.
 
 1. `aws organizations create-account --email "username+cli-test@redhat.com" --account-name "username-cli-test" --profile=orgtest`
-
 2. `aws organizations list-accounts --profile=orgtest | jq '.[][] | select(.Name=="username-cli-test")'`
+3. `aws sts assume-role --role-arn arn:aws:iam::<ID>:role/OrganizationAccountAccessRole --role-session-name username-cli-test --profile=orgtest`
+
+# 2. The Custom Resources
+
+## 2.1. AccountPool CR 
+
+The AccountPool CR holds the information about the available number of accounts that can be claimed for cluster provisioning
+
+```yaml
+apiVersion: aws.managed.openshift.io/v1alpha1
+kind: AccountPool
+metadata:
+  name: example-accountpool
+  namespace: aws-account-operator
+  selfLink: /apis/aws.managed.openshift.io/v1alpha1/namespaces/aws-account-operator/accountpools/example-accountpool
```

Comment:
Food for thought: Should we give an example of a CR anyone can create, or give the CR as it looks after creation (as it is here)?

Reply:
The example CR's anyone can deploy are inside of the deploy directory, but in the future we should call this out in the docs 

> Food for thought: Should we give an example of a CR anyone can create, or give the CR as it looks after creation (as it is here)?

They are already present in the deploy directory, but we should note that in the doc some where. Maybe the next PR?

Reply:
The metadata is messy and distracts from the point of the CR imo. Each CR example should Name and Namespace, plus example spec. Maybe status.

Reply:
Yes I think the example here should contain a CR that you can apply not one that has been applied. It'd be confusing to the user to see fields that don't exist in the examples.